### PR TITLE
Updated th2 gradle plugin `0.1.1`

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
* Updated common: `5.14.0-dev`
* Updated cradle api: `5.4.1-dev`

### CODE REVIEW

**Review by OptimumCode — DISMISSED** [review]

File: build.gradle [hunk 1]
```diff
@@ -61,9 +61,9 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3'
 
     implementation "org.slf4j:slf4j-api"
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.2'
-    testImplementation 'org.mockito:mockito-junit-jupiter:5.10.0'
-    testImplementation 'org.mockito:mockito-core:5.10.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.3'
```
Comment (OptimumCode) [inline build.gradle:64]: note: looks like junit 5.11.0 is available

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:64]: I don't see 5.11.0 on the https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter

Comment (OptimumCode) [reply build.gradle:64]: I have just updated the version in th2 gradle plugin. Could you please try to update the version and if it is found by Gradle - commit the update?

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle]: the 5.11.0 version found, thank you.

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**